### ISSUE TITLE
DL3048 does not accept underscore and slash as valid char in LABEL key

### ISSUE THREAD

**Issue by mschmeitz** [hadolint/hadolint#1035]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
```bash
LABEL org.opencontainers.image.authors="Me"   # passed DL3048 check
LABEL my.label/with_underscore="Illegal underscore and slash"   # should pass the DL3048 check
```

### Actual behavior
Slashes and Underscores are not accepted in the LABEL key.
The following definition should pass but is reported as a failure:
```bash
LABEL my.label/with_underscore="Illegal underscore and slash"  
```
### Steps to reproduce the behavior
See above

This is expected, since the code of DL3048 defines the allowed characters.
See https://hackage.haskell.org/package/hadolint-2.12.0/docs/src/Hadolint.Rule.DL3048.html#containsIllegalChar
specifically the  line:
```bash
validChars = ['0'..'9'] ++ ['a' .. 'z'] ++ ['.', '-']
```
PR: #1034 

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
2.12.0
```

**Additional environment details (OS, stack version, etc.)**
